data/SftpReadDemoData.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<entity-facade-xml type="demo">
+    <moqui.service.message.SystemMessageType
+            systemMessageTypeId="AnalyticsDataSync"
```

Comment:
Let's rename it to "readOLAPData"

Reply:
Rename to `ReadOLAPData`. I have noticed a pattern where IDs are written in PascalCase